dev/try_builders.json

```diff
@@ -532,13 +532,6 @@
       "enabled": true,
       "run_if": ["dev/**", "packages/flutter_tools/**", "bin/**"]
     },
-    {
-      "name": "Mac dart_plugin_registry_test",
```

Comment:
> Do you know how (or whether) to remove the "Mac dart_plugin_registry_test" presubmit check for this?

I would have thought this would have done it...

Reply:
oh, I wonder if uses the JSON from HEAD?

Reply:
@keyonghan why would it show up in this PR as a pre-submit check if it was removed from try_builders?

Reply:
It should be reflected in the PR immediately.
There is some recent change about how builders are retrieved. /cc @CaseyHillers any idea?

Reply:
I just edited the `try_builders.json` in https://github.com/flutter/flutter/pull/78888 and the two tests I added showed up...

Reply:
Was the try_builders.json change here in the first commit change of this PR? I am guessing if it was triggered first, then it will remain in this check run set. Though later changes only trigger other tests.